Prime_Number/Prime_Number.js

```diff
@@ -0,0 +1,33 @@
+//Description: Function to find if a given number is prime or not
+//Expected Output: returns true or false
+
+const isPrime = (num) => {
+    //Numbers less than 2 are not prime numbers (including negatives)
+    if (num < 2) {
+        return false;
+    }
+    //start finding factors for num , if only 2 factors then it's prime, else not
+    let i = 1, factors = 0;
+    while (i <= num) {
+        let rem = num % i; //check the remiander is zero or not
+        if (rem === 0) {
+            factors += 1;
+        }
+        i++;
+    }
+    if (factors === 2) { //exactly two factors makes it prime
+        return true;
+    }
+    else {  //else it not a prime number
+        return false;
+    }
+}
+
+//I/P and O/P Examples
```

Comment:
Can you remove the true/false from comment and add an output section at end of file for this inputs as multi line comment.